check_synology.py

```diff
@@ -123,7 +124,7 @@ def exitCode():
     perfdata = '|'
     for i in range(1,256,1):
         storage_name = snmpget('1.3.6.1.2.1.25.2.3.1.3.' + str(i))
-        if storage_name.startswith("/volume"):
+        if storage_name.startswith("/volume") and not re.match("/volume.+/@docker.+", storage_name) and not re.match("/volume.+/@docker", storage_name) :
```

Comment:
I would suggest using ```*``` instead of ```+``` in the regex resulting in:
```if storage_name.startswith("/volume") and not re.match("/volume.+/@docker.*", storage_name):```
There is no need to add another regex check if the previous can check both scenarios.

Reply:
Maybe we could also eliminate the first check ```if storage_name.startswith("/volume")``` if we use ```/volume(?!.+/@docker.*)``` as the regular expression.
Can you also validate this?

Reply:
> I would suggest using `*` instead of `+` in the regex resulting in:
> `if storage_name.startswith("/volume") and not re.match("/volume.+/@docker.*", storage_name):`
> There is no need to add another regex check if the previous can check both scenarios.

This one Works.

> Maybe we could also eliminate the first check `if storage_name.startswith("/volume")` if we use `/volume(?!.+/@docker.*)` as the regular expression.
> Can you also validate this?

This one two.
Will update my pull request with this change.

---------------
a bit offtopic, im new github and don't know where to put those questions : 

1.
i just got a Failed drive, DSM shows Status 'Warning'. The Status Check reports `System Status: Normal` and reports as 'OK'
Intended Behavior ? 
2.
memory check only considers really "free" memory as free. Synologys resource monitor considers free + cached memory as free. Is this a design decision?

Reply:
Thanks for contributing!

Regarding your 2 "offtopic" questions please open 2 separat issues where we can discuss these topics